frontend/services/entities/mdm.ts

```diff
@@ -324,13 +338,51 @@ const mdmService = {
     )}`;
     return sendRequest("DELETE", path);
   },
+
   getCommandResults: (
     command_uuid: string
   ): Promise<IGetMdmCommandResultsResponse> => {
     const { COMMANDS_RESULTS: MDM_COMMANDS_RESULTS } = endpoints;
     const url = `${MDM_COMMANDS_RESULTS}?command_uuid=${command_uuid}`;
     return sendRequest("GET", url);
   },
+
+  getSetupExperienceScript: (
+    teamId: number
```

Comment:
```suggestion
    teamId?: number
```

On the backend, no team is implemented as `GET /setup_experience/script` and teams are implemented as `GET /setup_experience/script?team_id=1`

Reply:
ah ok thanks. I will update this.